pkg/util/merr/utils.go

```diff
@@ -185,6 +188,9 @@ func OldCodeToMerr(code commonpb.ErrorCode) error {
 	case commonpb.ErrorCode_SegmentNotFound:
 		return ErrSegmentNotFound
 
+	case commonpb.ErrorCode_MetaFailed:
```

Comment:
why could we map this code?

Reply:
Only flush manager checks this error, and it's actually channel not found

Reply:
But it make no sense to convert here without any comment, it looks like a "error conversion"